providers/in-process-flagd/provider.go

```diff
@@ -0,0 +1,644 @@
+package flagd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/open-feature/flagd/core/pkg/eval"
+	"github.com/open-feature/flagd/core/pkg/logger"
+	"github.com/open-feature/flagd/core/pkg/runtime"
+	"github.com/open-feature/flagd/core/pkg/store"
+	"github.com/open-feature/flagd/core/pkg/sync"
```

Comment:
really great that we can use these libs. The implementation is surprisingly small.